fbuapp/View Controllers/RegisterViewController.m

```diff
@@ -9,96 +9,106 @@
+    self.navigationItem.leftBarButtonItem = [[UIBarButtonItem alloc] initWithTitle:@"Cancel" style:UIBarButtonItemStylePlain target:self action:@selector(didCancel)];
+    
     if ([FBSDKAccessToken currentAccessToken]) {
-        
-        //FBSDK --> gets user's name and user ID
         [FBSDKProfile loadCurrentProfileWithCompletion:^(FBSDKProfile *profile, NSError *error) {
             if (profile){
-                //access to user's email --> error: only getting "email" and not the actual value
+                //Get access to user's facebook email
                 [[[FBSDKGraphRequest alloc] initWithGraphPath:@"me" parameters:@{@"fields": @"email"}]
                  startWithCompletionHandler:^(FBSDKGraphRequestConnection *connection, id result, NSError *error) {
                      if (!error) {
                          NSLog(@"user:%@", result);
-                         //setting property to result dictionary value
                          self.emailField.text = result[@"email"];
                      }
                  }];
-    
+                
                 //get user's first and last name
-                //setting my created properties to currentProfile properties
                 self.firstNameField.text = profile.firstName;
                 self.lastNameField.text = profile.lastName;
-                
                 //test
                 NSLog(@"Hello %@!", profile.firstName);
                 
-                //welcome user to app in navigation bar
+                //Welcome user
                 self.navigationItem.title = [NSString stringWithFormat:@"Welcome %@", profile.firstName];
-                
-                //display users's facebook profile picture
-                FBSDKProfilePictureView *profilePictureView = [[FBSDKProfilePictureView alloc] init];
-                //manual placement of profile page
-                profilePictureView.frame = CGRectMake(137, 103, 100, 100);
-                //show profile of current user logged in
-                profilePictureView.profileID = [[FBSDKAccessToken currentAccessToken] userID];
-                [self.view addSubview:profilePictureView];
             }
         }];
     }
+    
+    // Shadow and Radius of signupButton
+    self.signupButton.layer.shadowColor = [[UIColor colorWithRed:0 green:0 blue:0 alpha:0.25f] CGColor];
+    self.signupButton.layer.shadowOffset = CGSizeMake(0, 2.0f);
+    self.signupButton.layer.shadowOpacity = 1.0f;
+    self.signupButton.layer.shadowRadius = 0.0f;
+    self.signupButton.layer.cornerRadius = 4.0f;
+    
+    //make profile image circular
+    self.profilePictureView.layer.cornerRadius = self.profilePictureView.frame.size.width / 2;
+    self.profilePictureView.clipsToBounds = YES;
+    //add border to profile image
+    self.profilePictureView.layer.borderWidth = 3.0f;
+    self.profilePictureView.layer.borderColor = [UIColor whiteColor].CGColor;
```

Comment:
This code is very similar to that in ProfileViewController. Perhaps make some helper methods to be used in both.